dashactivity.php

```diff
@@ -124,207 +115,182 @@ public function hookDashboardData($params)
+                        '<i class="icon-circle" style="color:' . self::$colors[2] . '"></i> Direct Traffic' => round($visits / 4),
+                    ],
+                ],
+                'data_chart' => [
+                    'dash_trends_chart1' => [
                         'chart_type' => 'pie_chart_trends',
-                        'data' => array(
-                            array('key' => 'prestashop.com', 'y' => round($visits / 2), 'color' => self::$colors[0]),
-                            array('key' => 'google.com', 'y' => round($visits / 3), 'color' => self::$colors[1]),
-                            array('key' => 'Direct Traffic', 'y' => round($visits / 4), 'color' => self::$colors[2])
-                        )
-                    )
-                )
-            );
+                        'data' => [
+                            ['key' => 'prestashop.com', 'y' => round($visits / 2), 'color' => self::$colors[0]],
+                            ['key' => 'google.com', 'y' => round($visits / 3), 'color' => self::$colors[1]],
+                            ['key' => 'Direct Traffic', 'y' => round($visits / 4), 'color' => self::$colors[2]],
+                        ],
+                    ],
+                ],
+            ];
         }
 
-        $gapi = Module::isInstalled('gapi') ? Module::getInstanceByName('gapi') : false;
-        if (Validate::isLoadedObject($gapi) && $gapi->isConfigured()) {
-            $visits = $unique_visitors = $online_visitor = 0;
-            if ($result = $gapi->requestReportData('', 'ga:visits,ga:visitors', Tools::substr($params['date_from'], 0, 10), Tools::substr($params['date_to'], 0, 10), null, null, 1, 1)) {
-                $visits = $result[0]['metrics']['visits'];
-                $unique_visitors = $result[0]['metrics']['visitors'];
-            }
-        } else {
-            $row = Db::getInstance(_PS_USE_SQL_SLAVE_)->getRow('
-						SELECT COUNT(*) as visits, COUNT(DISTINCT `id_guest`) as unique_visitors
-						FROM `'._DB_PREFIX_.'connections`
-						WHERE `date_add` BETWEEN "'.pSQL($params['date_from']).'" AND "'.pSQL($params['date_to']).'"
-						'.Shop::addSqlRestriction(false)
-                    );
-            extract($row);
-        }
+        $visits = $unique_visitors = 0;
+        $row = Db::getInstance((bool) _PS_USE_SQL_SLAVE_)->getRow('
+            SELECT COUNT(*) as visits, COUNT(DISTINCT `id_guest`) as unique_visitors
+            FROM `' . _DB_PREFIX_ . 'connections`
+            WHERE `date_add` BETWEEN "' . pSQL($params['date_from']) . '" AND "' . pSQL($params['date_to']) . '"
+            ' . Shop::addSqlRestriction(false)
+        );
+        extract($row);
 
         // Online visitors is only available with Analytics Real Time still in private beta at this time (October 18th, 2013).
-        // if ($result = $gapi->requestReportData('', 'ga:activeVisitors', null, null, null, null, 1, 1))
         // $online_visitor = $result[0]['metrics']['activeVisitors'];
```

Comment:
```suggestion
```

Reply:
The whole comment block can be removed no?

Reply:
Fixed